subsys/bluetooth/audio/stream.c

```diff
@@ -50,9 +50,6 @@ void bt_audio_stream_attach(struct bt_conn *conn,
 	}
 }
 
-#define IN_RANGE(_min, _max, _value) \
-	(_value >= _min && _value <= _max)
-
```

Comment:
Aren't you also missing the change to `bt_audio_valid_stream_qos()` in this file, since that's right now relying on a different order of the macro arguments?

Reply:
Wow, thanks for seeing that!